### ISSUE TITLE
iOS 14

### ISSUE THREAD

**Issue by Progamer78** [ryannair05/ApolloAPI#2]
Think you could make this have a rooted option also? Trying to download on Havoc but it was it is ios15+ and rootless.

**Comment by hmm-norah** [comment]
Seconding this, last compatible version for 14.3 seems to be 1.14.26.

**Comment by psaidwid** [comment]
Yes please this ^. iOS 14.3, noticed the tweak on the new section.
